lib/tailwindcss/compressor.rb

```diff
@@ -23,4 +23,12 @@ def call(input)
       input[:data]
     end
   end
+
+  private
+
+  def files_with_class_names
+    default_files = Rails.root.glob("app/views/**/*.*") + Rails.root.glob("app/helpers/**/*.rb")
```

Comment:
Don't think we need to set a default twice? Since you're doing it in the initializer, that should be enough.